src/overlays/actors/ovl_En_Baba/z_en_baba.c

```diff
@@ -311,212 +334,219 @@ void func_80BA8DF4(EnBaba* this, GlobalContext* globalCtx) {
         Math_SmoothStepToS(&this->torsoRot.y, 0, 4, 0x3E8, 1);
     }
 
-    SubS_FillLimbRotTables(globalCtx, this->unk_302, this->unk_326, ARRAY_COUNT(this->unk_302));
+    SubS_FillLimbRotTables(globalCtx, this->limbRotTableY, this->limbRotTableZ, ARRAY_COUNT(this->limbRotTableY));
 
-    if (this->unk_40A & 2) {
-        func_80BA8C90(this, globalCtx);
+    if (this->flags & ENBABA_VISIBLE) {
+        EnBaba_UpdateCollider(this, globalCtx);
     }
 }
 
-s32 func_80BA8F88(EnBaba* this, GlobalContext* globalCtx, ScheduleResult* arg2) {
-    u16 sp26 = (u16)(gSaveContext.save.time - 0x3FFC);
-    u16 temp;
-    u8 sp23 = ENBABA_GET_3F00(&this->actor);
+s32 EnBaba_InitTimePath(EnBaba* this, GlobalContext* globalCtx, ScheduleResult* scheduleResult) {
+    u16 now = SCHEDULE_TIME_NOW;
+    u16 startTime;
+    u8 path = ENBABA_GET_PATH(&this->actor);
```

Comment:
```suggestion
    u8 pathIndex = ENBABA_GET_PATH_INDEX(&this->actor);
```
I think `path` and `pathIndex` should be distinguished